lib/typinggame_server/interactors/players/index.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
```

Comment:
Any reason for this? We don't have any strings in this file.

Reply:
@jgayfer fixed